openff/interchange/interop/openmm/__init__.py

```diff
@@ -36,7 +38,7 @@ def to_openmm_system(
     combine_nonbonded_forces: bool = False,
     add_constrained_forces: bool = False,
     ewald_tolerance: float = 1e-4,
-    hydrogen_mass: float = 1.007947,
+    hydrogen_mass: PositiveFloat = 1.007947,
```

Comment:
I know we have the annotation and all, but what if I passed a negative value here? Is there anything at runtime that would actually stop me from creating negatively massed hydrogens with super heavy atoms?

Reply:
Yep - slightly buried in the logic so it doesn't stand out in the diff: https://github.com/openforcefield/openff-interchange/pull/1065/files#diff-80d8de9a0e22631e9558b73e53bb5e90a13dd20d0e8cb9fa26f25c95622d4745R222-R228